src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Encapsulates the idea of a task, set as abstract
+ * to prevent instantiation
+ */
+package duke.task;
+
+public abstract class Task {
+
+    // String constants for labelling state of task
+    protected static final String CROSS = "✗";
+    protected static final String CHECK = "✓";
+
+    // instance fields
+    protected String desc; // description of task
+    protected boolean isDone; // state of task
+
+    // constructors
+    Task(String desc) {
+        this.desc = desc;
+    }
+    Task(String desc, boolean isDone) {
+        this.desc = desc;
+        this.isDone = isDone;
+    }
+
+    // accessors for labels, descriptions and status of task
+    public String getStateLabel() {
```

Comment:
Missing javadocs for public methods.
 This issue also appeared in multiple other areas as well.